### ISSUE TITLE
Automatic1111 changing model

### ISSUE THREAD

**Issue by Jeremy-Developer-Page** [open-webui/open-webui#5134]
# Bug Report
If you try to change the template to be used to generate images with the Automatic1111 API it will not be done and the last template loaded by the Automatic1111 UI will be kept. Besides that if the generation takes more than a minute the image is not shown in the Openwebui ui. To solve this problem you can use the xformers library on automatic1111 which significantly decreases the generation time

## Installation Method
Docker Open WebUI/Ollama
Locally Automatic1111

## Environment
Open WebUI 0.3.16
Ollama 0.3.9
Automatic1111 1.10.1

Linux (OpenWebUI and Ollama)
Windows (Automatic1111)
Browser(MS Edge, Edge for iOS, Safari)

**Confirmation:**
- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Automatic 1111 changing model used
## Actual Behavior:

Nothing changes
## Reproduction Details
Go in admin stettings, change the model and see that Atuomatic1111 doesn't change it (if you download the png and go to png info of automatic1111 you'll see that the model is still the same

## Note
Automatic1111 offers much more customization and you could implement for example default negative prompts, the part related to hires, cfg scale, sampling method to be used, type of upscaling to be used. etc. 

In short there is a lot to improve for this application and it would be enough to implement the APIs made available or at least make them accessible to the user so that they can be changed to improve the user experience

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!
